cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -317,6 +317,9 @@ instance Era era => ToObject (UtxowPredicateFailure era) where
              , "txBodyMetaDataHash" .= txBodyMetaDataHash
              , "fullMetaDataHash" .= fullMetaDataHash
              ]
+  toObject _verb InvalidMetaData =
+    mkObject [ "kind" .= String "InvalidMetaData"
+             ]
```

Comment:
Note that this new constructor can also be sent to local client that submit an invalid transaction (because of invalid metadata). If such a client is not running 1.21, it will choke on the message ...